src/common/schedulers/slurm_commands.py

```diff
@@ -317,7 +317,7 @@ def set_nodes_down_and_power_save(node_list, reason):
     set_nodes_power_down(node_list, reason=reason)
 
 
-def get_nodes_info(nodes, command_timeout=5):
+def get_nodes_info(nodes, command_timeout=15):
```

Comment:
can we leave a trace on why it was necessary to increase he timeout? Also if I'm not wrong Slurm commands timeout after 10 seconds

nit: it would be nice to set this default timeout as a module constant so that we can change it in a single place when necessary

Reply:
oh you are right it was 1 min. Thanks